File: 5KYU/RGB to Hex Conversion/solution.fs
let clamp (min : int) (max : int) (n : int) : int =
    if n < min then min
    else if n > max then max
    else n

let rgb r g b =
    let r = clamp 0 255 r
    let g = clamp 0 255 g
    let b = clamp 0 255 b
    
    r.ToString("X2") + g.ToString("X2") + b.ToString("X2")
    

File: 5KYU/RGB to Hex Conversion/solution.kt
fun clamp(min: Int, max: Int, n: Int): Int {
    if(n < min) {
        return min;
    } else if (n > max) {
        return max;
    }
    return n;
}

fun rgb(r: Int, g: Int, b: Int): String {
  var rc = clamp(0, 255, r);
  var gc = clamp(0, 255, g);
  var bc = clamp(0, 255, b);
  
  return (rc.toString(16).padStart(2, '0') + gc.toString(16).padStart(2, '0') + bc.toString(16).padStart(2, '0')).uppercase();
}